### ISSUE TITLE
Support 2160p at 60Hz?

### ISSUE THREAD

**Issue by mcaskey** [Floris497/mac-pixel-clock-patch#22]
Will this add support for 2160p at 60Hz on a 2013 rMBP 15" with El Cap? Thanks!

**Comment by Floris497** [comment]
Depends on the configuration, but I'm quite sure it does. Give it a shot I would say!

**Comment by mcaskey** [comment]
Will do!

**Comment by bumper314** [comment]
Just a report: No luck for me on my 2013 rMBP 15" running 10.11.3.  I used the v2 script and patched both IOKit and NVIDIA.  I still get 2160p @ 30Hz over HDMI.

**Comment by Floris497** [comment]
Yes! Completely true! No MacBook till this day has HDMI2.0 but at tops HDMI1.4a which at top is capable of 4k@30Hz. An adapter from mDP ro HDMI2.0 with help. Make 100% sure it is HDMI2.0 or up adapters claiming 4k are mostly 30Hz. I think the only one now is from club3d, as far as I know.

**Comment by bumper314** [comment]
Thanks!
